blessed/src/main/java/com/welie/blessed/BluetoothPeripheralManager.java

```diff
@@ -607,6 +627,15 @@ private void cancelConnection(@NotNull final BluetoothDevice bluetoothDevice) {
         return bluetoothManager.getConnectedDevices(BluetoothGattServer.GATT);
     }
 
+    private BluetoothDevice getDeviceForCentral(@NotNull final BluetoothCentral bluetoothCentral) {
```

Comment:
This entire method is unnecessary. Can we replaced by:

bluetoothAdapter.getRemoteDevice(bluetoothCentral.getAddress())

Reply:
Removed method and changed usage to your above code snippet